adops/setting-up-prebid-multiformat-in-dfp.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Setting up Prebid Multiformat in DFP
+head_title: Setting up Prebid Multiformat in DFP
+description: Setting up Prebid Multiformat in DFP
+pid: 3
+hide: false
+top_nav_section: adops
+nav_section: tutorials
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Setting up Prebid Multiformat in DFP
+{: .no_toc}
+
+This page shows how to set up your ad server so that you can serve multiformat ads.
+
+Multiformat ads allow you to declare multiple media types on a single ad unit.  For example, you can set up one ad on the page that could show a banner, native, or outstream video ad, depending on which had the highest bid.
+
+{: .alert.alert-info :}
+For instructions, on how to set up multiformat ads from the engineering side, see [Show Multiformat Ads with Prebid.js]({{site.baseurl}}/dev-docs/show-multiformat-ads.html).
+
+* TOC
+{: toc }
+
+## Step 1. Add an Order
+
+In DFP, create a new order for each of your header bidding partners.
+
+Repeat this step and step 2 when you are adding a new partner.
+
+## Step 2. Add Line Items and Creatives for each Media Type
+
+Because of the way multiformat support works in Prebid.js, for each price bucket, you will need to set up line items for each of the following formats:
```

Comment:
As written, it seems like we're assuming that the user will be supporting banner, outstream and native on a single ad unit, but that might not be the case.

We should basically be noting that multi-format ad units including native require at least two distinct sets of LIs/ ad server Creatives.

Reply:
Updated the language in c857214